src/utils/utils.go

```diff
@@ -123,3 +124,17 @@ func OpenBrowser(url string) {
 
 	log.Printf("Please open %s", url)
 }
+
+func ParseProxyUrl(proxyAddr string) (*url.URL, error) {
```

Comment:
Considering my previous comment about this check being used in main I think it's better to move this function straight to src/jobs/rawnet.go

Reply:
It's was used in main.go for global proxy url validation and related not only to the rawnet job.